docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -378,6 +380,10 @@ Additionally it is possible to set:
   `<Response_Header_1, ..., Response_Header_n>` to specify headers to pass to backend once authentication request completes.
 * `nginx.ingress.kubernetes.io/auth-request-redirect`:
   `<Request_Redirect_URL>`  to specify the X-Auth-Request-Redirect header value.
+* `nginx.ingress.kubernetes.io/auth-cache-key`:
+  `<Cache_Key>` this enables caching for auth requests. specify a lookup key for auth responses. e.g. `$remote_user$http_authorization`.
```

Comment:
Should we make a note that this will be hashed and encoded?

Reply:
I think the implementation details are not important - the mechanics are tho!

I appended this:
> Each server and location has it's own keyspace. Hence a cached response is only valid on a per-server and per-location basis